api/spec/v1.yaml

```diff
@@ -1226,6 +1226,15 @@ components:
         success:
           type: boolean
           description: Whether this transaction successfully executed.
+        code:
+          type: integer
+          description: The status code of a failed call.
```

Comment:
We do refer to "call" and "transaction" in this doc. 🤔

Reply:
I think of "call" as a specific type of tx (especially in evm lingo, with calls and queries), and would prefer "transaction" here. We _could_ update other uses of "call", I see there aren't many. WDYT?

Reply:
Agree. I think I'll preserve existing references to "call" because they are verbs and point to more internals.

Reply:
Update. Opted for "transaction" in this PR's changes, but preserved the existing references.